db/migrate/20200607113340_create_posts.rb

```diff
@@ -0,0 +1,16 @@
+class CreatePosts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
```

Comment:
I think the title length should have a character limit to prevent display issues (especially on mobile)

Reply:
Added a max of 100 chars